package.json

```
@@ -16,8 +16,9 @@
       "import": "./dist/module.mjs",
       "require": "./dist/module.cjs"
     },
-    "./transformers": "./dist/runtime/transformers/index.mjs",
-    "./transformers/*": "./dist/runtime/transformers/*.mjs"
+    "./transformers": "./dist/runtime/transformers/index.js",
+    "./transformers/*": "./dist/runtime/transformers/*.js",
+    "./dist/runtime/types": "./dist/runtime/types/index.js"
```

We could also add something like this, and document instead importing/augmenting `@nuxt/content/types`:

```suggestion
    "./types": "./dist/runtime/types/index.js",
    "./dist/runtime/types": "./dist/runtime/types/index.js"
```